Documentation/ExtensionArchitecture/FileStructure/ExtTables.rst

```diff
@@ -113,27 +110,3 @@ module:
           'labels' => 'LLL:EXT:my_extension/Resources/Private/Language/locallang_statistics.xlf',
       ]
    );
-
-.. index:: Extension development; allowTableOnStandardPages
-.. _extension-configuration-files-allow-table-standard:
-
-Allowing a tables records to be added to Standard pages
```

Comment:
I would keep a hint and link to t3tca:ctrl-security-ignorePageTypeRestriction here